.github/fabricbot.json

```diff
@@ -1,1182 +0,0 @@
-          ]
-        },
-        {
-          "labels": [
-            "area-Extensions-HttpClientFactory"
-          ],
-          "mentionees": [
-            "dotnet/ncl"
-          ]
-        },
-        {
-          "labels": [
-            "area-Extensions-Logging"
-          ],
-          "mentionees": [
-            "dotnet/area-extensions-logging"
-          ]
-        },
-        {
-          "labels": [
-            "area-Extensions-Options"
-          ],
-          "mentionees": [
-            "dotnet/area-extensions-options"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.Net.Quic"
-          ],
-          "mentionees": [
-            "dotnet/ncl"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.Formats.Cbor"
-          ],
-          "mentionees": [
-            "dotnet/area-system-formats-cbor"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.Formats.Asn1"
-          ],
-          "mentionees": [
-            "dotnet/area-system-formats-asn1"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.Text.Json"
-          ],
-          "mentionees": [
-            "dotnet/area-system-text-json"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.Memory"
-          ],
-          "mentionees": [
-            "dotnet/area-system-memory"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.Diagnostics.Activity"
-          ],
-          "mentionees": [
-            "dotnet/area-system-diagnostics-activity"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.IO"
-          ],
-          "mentionees": [
-            "dotnet/area-system-io"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.Diagnostics.Process"
-          ],
-          "mentionees": [
-            "dotnet/area-system-diagnostics-process"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.Console"
-          ],
-          "mentionees": [
-            "dotnet/area-system-console"
-          ]
-        },
-        {
-          "labels": [
-            "area-System.IO.Compression"
-          ],
-          "mentionees": [
-            "dotnet/area-system-io-compression"
-          ]
-        }
-      ],
-      "replyTemplate": "Tagging subscribers to this area: ${mentionees}\nSee info in [area-owners.md](https://github.com/dotnet/runtime/blob/main/docs/area-owners.md) if you want to be subscribed.",
-      "enableForPullRequests": true
-    },
-    "disabled": false
-  },
-  {
-    "taskSource": "fabricbot-config",
-    "taskType": "trigger",
-    "capabilityId": "IssueResponder",
-    "subCapability": "IssuesOnlyResponder",
-    "version": "1.0",
-    "config": {
-      "taskName": "Add untriaged label to new/reopened issues without a milestone",
```

Comment:
This task didn't get ported over, presumably because it used the `removedFromMilestone` action which is not supported.